client/app/components/faq/faqlist.js

```diff
@@ -35,94 +30,97 @@ class FaqList extends Component {
 
 	componentDidMount() {
 		this.fetchData()
+			// eslint-disable-next-line no-console
 			.then(() => console.log("Finished mounting!"))
+			// eslint-disable-next-line no-console
 			.catch(err => console.error(err));
 	}
 
 	componentWillReceiveProps(nextProps) {
 		// Avoiding refresh if possible
-	  if (nextProps.currentQuestions !== this.state.currentQuestions) {
-		 this.setState({ currentQuestions: nextProps.currentQuestions });
-	  }
+		if (nextProps.currentQuestions !== this.state.currentQuestions) {
+			this.setState({ currentQuestions: nextProps.currentQuestions });
+		}
 	}
 
-	render() {
-		return (
-			<ScrollView
-				style={FaqStyles.faqPadding, {backgroundColor: 'white'}}
-				refreshControl={
-					<RefreshControl
-						refreshing={this.state.isRefreshing}
-						onRefresh={this._onRefresh.bind(this)}
-					/>
-				}
-			>
-				<FlatList
-					data={this.state.currentQuestions}
-					renderItem={this._renderListItem}
-					extraData={this.state}
-					keyExtractor={this._keyExtractor}
-					ListHeaderComponent={this._renderHeader}
-				/>
-			</ScrollView>
-		);
+	onRefresh = () => {
+		this.setState({isRefreshing: true});
+		this.fetchData().then(() => {
+			this.setState({isRefreshing: false});
+		});
 	}
 
 	fetchData() {
 		return fetch(`${API_URL}/list`)
 			.then(response => response.json())
 			.then(faqList => this.props.onFAQLoaded(faqList))
+			// eslint-disable-next-line no-console
 			.catch(err => console.error(err));
 	}
 
-	_onRefresh() {
-		this.setState({isRefreshing: true});
-		this.fetchData().then(() => {
-			this.setState({isRefreshing: false});
-		});
-	}
-
-	_renderListItem = ({ item, index }) => {
+	renderListItem({ item, index }) {
 		let rowBg = index % 2 == 1 ? FaqStyles.faqListItemOdd : null;
+		
+		const icon = (
+			<Icon
+				name="chevron-thin-right"
+				size={30}
+				color="#787878"
+				style={{marginTop: 5}}
+			/>
+		);
+
 		return (
 			<ListItem
-				containerStyle={FaqStyles.faqListItem, rowBg}
+				containerStyle={{...FaqStyles.faqListItem, rowBg}}
 				titleStyle={FaqStyles.faqListItemText}
 				key={item.id}
 				title={item.question}
-				onPress={() => this._renderFaqDetails(item, index)}
-				rightIcon={<Icon
-								name={'chevron-thin-right'}
-								size={30}
-								color={'#787878'}
-								style={{marginTop: 5}}
-						  />}
+				onPress={this.renderFaqDetails(item, index)}
+				rightIcon={icon}
 			/>
 		);
 	}
 
-	_renderHeader = () => {
+	renderHeader = () => {
```

Comment:
I don't like using arrow functions to circumvent binding this on functions (especially cause it sometimes messes with react native erroring) so I'm going to look into using an [autobind decorator](https://github.com/andreypopp/autobind-decorator)